DistributionPackages/Neos.MarketPlace/Resources/Private/Fusion/FusionObjects/Package/PackageReadme.fusion

```diff
@@ -1,19 +1,26 @@
-prototype(Neos.MarketPlace:PackageReadme) < prototype(Neos.Fusion:Template) {
-    templatePath = 'resource://Neos.MarketPlace/Private/Templates/FusionObjects/Package/PackageReadme.html'
+prototype(Neos.MarketPlace:PackageReadme) < prototype(Neos.Fusion:Component) {
+    readme = ${q(documentNode).children('readme').first()}
+    readmeSource = ${this.readme.property('source')}
```

Comment:
The property was renamed to `readmeSource`. I fixed it in the current implementation. See https://github.com/neos/Neos.NeosIo/pull/463